src/test/resources/README

```diff
@@ -0,0 +1,18 @@
+Describes files and how they can be used
```

Comment:
Like. I think it will be useful given the number of mock files we've got so far.